cmd/resetHeaderCheckCount.go

```diff
@@ -26,21 +26,27 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var blockNumber int
+var blockNumberFlagValue int
```

Comment:
Wondering if we might be able to simplify things by declaring this an `int64` and using `Int64VarP` to parse the flag?

Reply:
ah, great call!